tests/models/snapshot.py

```diff
@@ -37,7 +37,8 @@ def test_snapshot_filepath(tmpdir):
 def test_snapshot_reference_non_existant():
     """Tests snapshots for non-existant references."""
     snapshot = Snapshot('bogus')
-    assert snapshot.reference is None
+    with pytest.raises(ValueError):
```

Comment:
This test is incorrect, it shouldn't raise `ValueError` (note we have a test for that a few lines above). I tried it locally and it should be adjusted to:
```python
def test_snapshot_reference_non_existant(tmpdir):
    """Tests snapshots for non-existant references."""
    reference_file = tmpdir.join('non-existant')
    snapshot = Snapshot('non-existant', data_dir=reference_file.dirname)
    assert snapshot.reference is None
```